Analysis/Tasks/PWGCF/o2femtodream/include/o2femtodream/FemtoDerived.h

```diff
@@ -0,0 +1,88 @@
+#include "AnalysisDataModel/PID/PIDResponse.h"
+
+namespace o2::aod
+{
+
+namespace femtodreamcollision
+{
+DECLARE_SOA_COLUMN(MultV0M, multV0M, float);
+DECLARE_SOA_COLUMN(Sphericity, sphericity, float);
+} // namespace femtodreamcollision
+DECLARE_SOA_TABLE(FemtoDreamCollisions, "AOD", "FEMTODREAMCOLS",
+                  o2::soa::Index<>,
+                  o2::aod::collision::PosZ,
+                  femtodreamcollision::MultV0M,
+                  femtodreamcollision::Sphericity);
+using FemtoDreamCollision = FemtoDreamCollisions::iterator;
+
+namespace femtodreamparticle
+{
+DECLARE_SOA_INDEX_COLUMN(FemtoDreamCollision, femtoDreamCollision);
+DECLARE_SOA_COLUMN(Pt, pt, float);
+DECLARE_SOA_COLUMN(Eta, eta, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_COLUMN(Cut, cut, uint64_t);
+DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);
+// debug variables
+DECLARE_SOA_COLUMN(Sign, sign, int8_t);
+DECLARE_SOA_COLUMN(TPCNClsFound, tpcNClsFound, uint8_t);
+DECLARE_SOA_COLUMN(TPCNClsCrossedRows, tpcNClsCrossedRows, uint8_t);
+DECLARE_SOA_DYNAMIC_COLUMN(TPCCrossedRowsOverFindableCls, tpcCrossedRowsOverFindableCls, //!
+                           [](int8_t tpcNClsFindable, int8_t tpcNClsCrossedRows) -> float {
+                             return (float)tpcNClsCrossedRows / (float)tpcNClsFindable;
+                           });
+
+} // namespace femtodreamparticle
+DECLARE_SOA_TABLE(FemtoDreamParticles, "AOD", "FEMTODREAMPARTS",
+                  o2::soa::Index<>,
+                  femtodreamparticle::FemtoDreamCollisionId,
+                  femtodreamparticle::Pt,
+                  femtodreamparticle::Eta,
+                  femtodreamparticle::Phi,
+                  femtodreamparticle::Cut,
+                  femtodreamparticle::TempFitVar);
+using FemtoDreamParticle = FemtoDreamParticles::iterator;
+
+DECLARE_SOA_TABLE(FemtoDreamDebugParticles, "AOD", "FEMTODEBUGPARTS",
+                  soa::Index<>,
+                  femtodreamparticle::FemtoDreamCollisionId,
+                  femtodreamparticle::Sign,
+                  femtodreamparticle::TPCNClsFound,
+                  track::TPCNClsFindable,
+                  femtodreamparticle::TPCNClsCrossedRows,
+                  track::TPCNClsShared,
+                  femtodreamparticle::TPCCrossedRowsOverFindableCls<track::TPCNClsFindable, femtodreamparticle::TPCNClsCrossedRows>,
+                  track::DcaXY,
+                  track::DcaZ,
+                  pidtpc_tiny::TPCNSigmaStoreEl,
+                  pidtpc_tiny::TPCNSigmaStorePi,
+                  pidtpc_tiny::TPCNSigmaStoreKa,
+                  pidtpc_tiny::TPCNSigmaStorePr,
+                  pidtpc_tiny::TPCNSigmaStoreDe,
+                  pidtof_tiny::TOFNSigmaStoreEl,
+                  pidtof_tiny::TOFNSigmaStorePi,
+                  pidtof_tiny::TOFNSigmaStoreKa,
+                  pidtof_tiny::TOFNSigmaStorePr,
+                  pidtof_tiny::TOFNSigmaStoreDe);
```

Comment:
I think you need the dynamic columns as well which expand the TPCNSigmaStoreEl to a usable value. I think you can simply reuse the definition, no copy paste of the code needed.